scripts/gha-setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# In order to set project env-vars, we must encrypt secrets
+# Using gh client allows us to set the secret without installing another
+# binary just to encrypt the secrets
+
+# Login GH client
+# GITHUB_ACCESS_TOKEN is injected when zero apply runs
+gh auth login --with-token <<EOF
```

Comment:
TODO: we will need to conditionally require `gh` binary as a pre-requisite if they pick `github action` as their CI Vendor